AnnService/inc/Core/Common/TruthSet.h

```diff
@@ -165,6 +165,7 @@ namespace SPTAG
                     exit(-1);
                 }
 
+                LOG(Helper::LogLevel::LL_Info, "Begin to generate truth for query(%d,%d) and doc(%d,%d)...\n", querySet->Count(), querySet->Dimension(), vectorSet->Count(), vectorSet->Dimension());
                 std::vector< std::vector<SPTAG::SizeType> > truthset(querySet->Count(), std::vector<SPTAG::SizeType>(K, 0));
                 std::vector< std::vector<float> > distset(querySet->Count(), std::vector<float>(K, 0));
 #pragma omp parallel for
```

Comment:
Unrelated to the larger change, but do you think it would be useful to track progress here like we do in SearchSequential?

Reply:
just add some logs to trace the progress of generate truth.